lib/game.rb

```diff
@@ -0,0 +1,100 @@
+require_relative 'player'
+require_relative 'board'
+
+class Game
+
+  attr_reader :player_1, :player_2, :board, :current_player
+
+  def initialize
+    @selected_token = nil
+    @player_1 = welcome_player("player_1")
+    @player_2 = welcome_player("player_2")
+    @board = Board.new()
+    @current_player = @player_1
+  end
+
+  def play_turn
+    location = get_location()
+    board.update_field_values(current_player.token, location)
+    current_player.add_cell(location)
+  end
+
+  def populate_display
```

Comment:
This populate_display method might better be in the Board class. Most of the logic inside here has something to do with the Board anyway. I suggest transferring this method in Board and then just invoking this method as @board.populate_display here in the Game class (perhaps as one of the lines in play_turn). - Single Responsibility Principle